iree/tools/iree-benchmark-module-main.cc

```diff
@@ -53,6 +53,10 @@ ABSL_FLAG(std::string, function_inputs_file, "",
           "Provides a file for input shapes and optional values (see "
           "ParseToVariantListFromFile in vm_util.h for details)");
 
+ABSL_FLAG(
+    int32_t, warmup_iterations, 3,
```

Comment:
this looks good; it would also be nice to have `warmup_min_time` and I would suggest only having `warmup_min_time` with a positive default value. Otherwise, if `warmup_iterations` has a positive default value, for workloads with a high latency, there's a risk of warming up unnecessarily long. I'd also suggest renaming to `warmup_min_iters`, since once there are 2 controls, neither will dictate the exact number of iterations.